### ISSUE TITLE
terraform-provider-oci_v4.80.0 plugin crash oci_core_route_table

### ISSUE THREAD

**Issue by comet528** [oracle/terraform-provider-oci#1614]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version
Terraform v1.2.2
on linux_amd64

### Affected Resource(s)

oci_core_route_table

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. 
# Please remove any sensitive information from configuration files before sharing them. 
```

### Panic Output
[WARN] Resource discovery finished with errors listed below:

INFO 2022/06/15 15:34:36.459379 Error discovering `oci_core_route_table` resources for export_OpenDataVCN: [ERROR] terraform import command failed for resource 'oci_core_route_table.export_OpenDataRouteTableViaNAT' at id 'ocid1.routetable.oc1.........': exit status 1

Error: Plugin did not respond

The plugin encountered an error, and failed to respond to the
plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more
details.

Stack trace from the terraform-provider-oci_v4.80.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x331f659]

goroutine 29 [running]:
github.com/terraform-providers/terraform-provider-oci/internal/service/core.(*CoreRouteTableResourceCrud).SetData(0xc001d90ff0)
	github.com/terraform-providers/terraform-provider-oci/internal/service/core/core_route_table_resource.go:350 +0x339
github.com/terraform-providers/terraform-provider-oci/internal/tfresource.ReadResource({0x58a39f0, 0xc001d90ff0})
	github.com/terraform-providers/terraform-provider-oci/internal/tfresource/crud_helpers.go:271 +0x124
github.com/terraform-providers/terraform-provider-oci/internal/service/core.readCoreRouteTable(0xc001cffb80, {0x5149420, 0xc000144660})
	github.com/terraform-providers/terraform-provider-oci/internal/service/core/core_route_table_resource.go:130 +0xd5
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x51718bc, {0x5983948, 0xc00020d500}, 0x24, {0x5149420, 0xc000144660})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:335 +0x178
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc001615260, {0x5983948, 0xc00020d500}, 0xc002ccb650, {0x5149420, 0xc000144660})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:624 +0x35b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc00000c498, {0x5983948, 0xc00020d500}, 0xc00020d540)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:575 +0x4bd
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadResource(0xc0004c22a0, {0x59839f0, 0xc001d90120}, 0xc002d42480)
	github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:298 +0x1fb
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x4e93d00, 0xc0004c22a0}, {0x59839f0, 0xc001d90120}, 0xc002d42420, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:344 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00014e8c0, {0x59b0e70, 0xc001a4a000}, 0xc003356200, 0xc0019b5170, 0x84963f0, 0x0)
	google.golang.org/grpc@v1.32.0/server.go:1194 +0xc8f
google.golang.org/grpc.(*Server).handleStream(0xc00014e8c0, {0x59b0e70, 0xc001a4a000}, 0xc003356200, 0x0)
	google.golang.org/grpc@v1.32.0/server.go:1517 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	google.golang.org/grpc@v1.32.0/server.go:859 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.32.0/server.go:857 +0x294

Error: The terraform-provider-oci_v4.80.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

 Any references to this resource have been replaced with hard coded values in generated configurations

### Steps to Reproduce

1. `terraform apply`

**Comment by rashik-bhasin** [comment]
Fix has been merged in v4.80.1 https://github.com/oracle/terraform-provider-oci/releases/tag/v4.80.1 . Please use the latest version and try again. LEt us know if the issue still persists.

Thanks!

**Comment by ravinitp** [comment]
closing this issue now
